WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/CustomerSearchUICommand.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+import Yosemite
+
+/// Implementation of `SearchUICommand` for Customer search.
+///
+final class CustomerSearchUICommand: SearchUICommand {
+
+    typealias Model = Customer
+    typealias CellViewModel = TitleAndSubtitleAndStatusTableViewCell.ViewModel
+    typealias ResultsControllerModel = StorageCustomer
+
+    var searchBarPlaceholder: String = Localization.searchBarPlaceHolder
+
+    var searchBarAccessibilityIdentifier: String = "customer-search-screen-search-field"
+
+    var cancelButtonAccessibilityIdentifier: String = "customer-search-screen-cancel-button"
+
+    var resynchronizeModels: (() -> Void) = {}
+
+    var onDidSelectSearchResult: ((Customer) -> Void)
+
+    private let siteID: Int64
+
+    init(siteID: Int64, onDidSelectSearchResult: @escaping ((Customer) -> Void)) {
+        self.siteID = siteID
+        self.onDidSelectSearchResult = onDidSelectSearchResult
+    }
+
+    func createResultsController() -> ResultsController<StorageCustomer> {
+        let storageManager = ServiceLocator.storageManager
+        let descriptor = NSSortDescriptor(keyPath: \StorageCustomer.customerID, ascending: false)
+        return ResultsController<StorageCustomer>(storageManager: storageManager, sortedBy: [descriptor])
```

Comment:
This will display `StorageCustomers` from Core Data for any site, not just the current one, which could lead to confusion for some merchants with multiple stores.

If you add a predicate specifying the siteID, that should resolve the issue. No need to block this PR since it's behind a feature flag.

Reply:
Thanks! I committed the changes https://github.com/woocommerce/woocommerce-ios/commit/a8dd77898ca4facaf06bf66d1d48acf7e692a846 with this one as we went through these in our pairing as was pretty quick 👍